set.mm

```diff
@@ -509765,111 +510466,380 @@ elements in a zero ring (see ~ 0ringirng ).  (Contributed by Thierry
+        RUVKUVLZAVVBEVHVLVVGEOUJUVMWFZUVNZAHIYKZHVUAVJVEVKAVAHIAVWTVXEVXJUVOUVP
+        ZHIVUAEVUAWLZUKYGWFAHUVQABVPZHVLZUTVPZHVLZVYEVYGUYTYHVEZVCZVYEVYGVUAYHV
+        EZVCZVKAVYFVRZVYHVRZVYJVYEVYGEYHVEZVCZVYLVYNVYEMVEZVYGMVEZVYIVCVYPMVEZV
+        YJVYPVYNIUYTVJVEZVYOVYIMEUYTVYEVYGUYTWLZUKVYTWLZVYNHIVYEAVYBVYFVYHVYCYI
+        ZAVYFVYHUVRZXKZVYNHIVYGWUCVYMVYHWHZXKZAIVYTMVQZVYFVYHAVUOWUHVXSAHVYTVKV
+        UOWUHXPVYAHVYTIMUVTWFYLZYIAMEUYTUWAVCVLZVYFVYHAMEUYTUWBVCVLWUJAEUYTMAEO
+        IUAGMVUBVVCUJUKUQVVQURVVGVXRUWCZUWDEUYTMUWEWFYIVYOWLZVYIWLUWFVYNVYQVYEV
+        YRVYGVYIVYNVYFVYQVYEVKWUDVYNBCDEFGHIJKLMNOPQRSVYETUAUBUCUDUEAVXQVYFVYHU
+        FYIZAVVEVYFVYHUGYIZACVWHVLZVYFVYHUHYIZUIUJUKULUMUNUOUPUQURUSWUEYJYLVYNV
+        YHVYRVYGVKZWUFVYNBCDEFGHIJKLMNOPQRSVYGTUAUBUCUDUEWUMWUNWUPUIUJUKULUMUNU
+        OUPUQURUSWUGYJYLUWIVYNVYPHVLZVYSVYPVKVYNHEUWGVEVLZVYFVYHWURAWUSVYFVYHAE
+        YMVLZVYBVUAYMVLWUSAEVXTUWHZVYCAVUAAEYNVLZHEUWQVEZVLZVUAYNVLAEOUJVVFUWJZ
+        AVUSEOHVUCUJVWAUSVWRVVGUWKZWVCHEVUAVYDWVCWLZUWLWPZUWMIHEUKUWNUWOYIWUDWU
+        FVYOHEVYEVYGWULUWRWTVYNBCDEFGHIJKLMNOPQRSVYPTUAUBUCUDUEWUMWUNWUPUIUJUKU
+        LUMUNUOUPUQURUSVYNIVYOEVYEVYGUKWULAWUTVYFVYHWVAYIWUEWUGUWSYJYLUWPVYOVYK
+        VYEVYGHYOVLZVYOVYKVKHVUSUWTZYCVUCUXAVCZUXBZYOUSVUSYOVLWVJYOVLWVLYOVLOVS
+        UXCVUSYOUXDWVJWVKYOUXEUXFUXGZHVYOEVUAYOVYDWULUXIYPUXHUXJUXKAVYEKVLZVYHV
+        RZVRZVYEVYGUYTYQVEZVCZVYEVYGEYQVEZVCZMVEZHWVPVYEVYRWVQVCWVRWWAWVPVYRVYG
+        VYEWVQWVPVYHWUQAWVNVYHUXLZWVPBCDEFGHIJKLMNOPQRSVYGTUAUBUCUDUEAVXQWVOUFW
+        IZAVVEWVOUGWIZAWUOWVOUHWIZUIUJUKULUMUNUOUPUQURUSWVPHIVYGAVYBWVOVYCWIWWB
+        XKZYJYLUXMWVPIVYTWVSWVQMVUEVJVEZEUYTVYEVYGWUAUKWUBWVPVYEKWWGAWVNVYHUXNA
+        KWWGVKWVOAKOVJVEZWWGAVVEKVWBYKKWWHVKUGVWBJKVWGUXSKVWBOJUBVWGYGUXOZAOVUE
+        VJAVVBOVUEVKVVGEOVHUJUXPWFZYRYSZWIUXQZWWFAWUHWVOWUIWIAMEUYTUXRVCVLZWVOW
+        UKWIWVSWLZWVQWLWWGWLZUXTUYAZWVPIHWVTMAIHMUYBWVOAUAIVUJHMVWSURUYCWIWVPVY
+        EWVSVUEWWGIEVYGUKVUEWLZWWNWWOAEUYDVLZWVOAEWVEUYEWIZWWLWWFUYFZUYGUYHWVPW
+        VRWWAWVTVYEVYGVUAYQVEZVCWWPWVPWVTHVLZWWAWVTVKWVPWWRWVDVYEWWGVLVYHWXBWWS
+        AWVDWVOWVFWIWWLWWBWWGWVCWVSHVUEEVYEVYGWWQWWNWWOWVGUYIUYJWVPBCDEFGHIJKLM
+        NOPQRSWVTTUAUBUCUDUEWWCWWDWWEUIUJUKULUMUNUOUPUQURUSWWTYJYLWVPWVSWXAVYEV
+        YGWVIWVSWXAVKWVPWVMHWVSEVUAYOVYDWWNUYKUYLYTUYMVUDWLWVIVUEVUAVGVEVKWVMHV
+        UEEVUAYOVYDWWQUYNYPAKWWHVUDVJVEWWIAOVUDVJAOVUEVUDWWJAHEUYTMVUEIVHVUFVUH
+        VXSVXTWWQUYOZYSYRYSWWKAVYEVYGVUDYHVEZVCZVYEVYGVUEYHVEZVCZVKWVNVYGKVLVRA
+        WXGWXEAWXFWXDVYEVYGAVUEVUDYHWXCYRYTUYQWIAWVBWWMUYTYNVLWVEWUKUYTMEUYPWPW
+        VHUYRAVUSEOHVUAVUCUJVWAUSVWRVVFVYDUYSYS $.
+    $}
+
+    $d A p q r x $.  $d A q x y $.  $d D p x $.  $d E p q r x $.  $d E y $.
+    $d F p q r x $.  $d F y $.  $d K p q r x $.  $d K y $.  $d L p r x $.
+    $d L y $.  $d M p q x $.  $d p ph x $.
+    $( The degree of an algebraic field extension is the degree of the minimal
```

Comment:
Maybe more verbose (as in the description of this PR):

> The final theorem in this PR is ~ algextdeg, which is a proof that the degree of an algebraic extension (noted [ L : K ] ) is the degree of the minimal polynomial M ( A ) , whereas L is the field generated by K and the algebraic element A .

For example: The degree of an algebraic field extension (noted [ L : K ] ) is the degree of the minimal polynomial M( A ) , whereas L is the field generated by K and the algebraic element A .

Reply:
Yes! I will first merge this, then attempt to add better descriptions in the next PR.

Reply:
Added in #4788